ios-oidc/Oidc/OLOidc.swift

```diff
@@ -205,4 +205,16 @@ public class OLOidc: NSObject {
             task.resume()
         }
     }
+    
+    @objc public func refreshAccessToken(callback: @escaping ((Error?) -> Void)) {
+        olAuthState.authState?.setNeedsTokenRefresh()
+        olAuthState.authState?.performAction(freshTokens: { (freshAccessToken, idToken, error) in
+            if error != nil {
+                callback(error)
+                return
+            }
+            self.olAuthState.authState = self.olAuthState.authState
```

Comment:
I'm a swift n00b. But why do you need to set `self.olAuthState.authState` to itself?

Reply:
Oh hehe good question. At this point some properties of the authState have changed. This is just to trigger the setter so that the keychain gets updated automatically and that the user doesn't have to handle that.